api_v3/lib/types/KalturaPlaybackSource.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * @package api
+ * @subpackage objects
+ */
+class KalturaPlaybackSource extends KalturaObject{
+
+	/**
+	 * @var string
+	 */
+	public $deliveryProfileId;
+    
+	/**
+	 * @var string
+	 */
+	public $format;
+
+	/**
+	 * @var string
+	 */
+	public $priority;
+
+	/**
+	 * @var KalturaStringArray
+	 */
+	public $protocols;
+
+	/**
+	 * @var KalturaStringArray
+	 */
+	public $flavors;
```

Comment:
How come flavors are strings? If these are flavor ids, rename the property name.